builtin/providers/aws/resource_aws_ecache.go

```diff
@@ -0,0 +1,256 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/aws-sdk-go/aws"
+	elasticcache "github.com/hashicorp/aws-sdk-go/gen/elasticache"
```

Comment:
Thanks for checking. I'm going to fix this inconsistency if we concluded discussion for naming.

This `elasticcache` alias in import is not needed to compile. This actually is due to my personal reason. My editor doesn't work well for its jump feature when jumping to `aws-sdk-go/aws`. It was hard to find interface of API during implementation if there was no this prefix, so I put it.

But now I can eliminate such prefixes and I'll do after the review.